server/src/main/java/io/crate/metadata/information/InformationSchemaTableDefinitions.java

```diff
@@ -67,9 +67,10 @@ public InformationSchemaTableDefinitions(InformationSchemaIterables informationS
         ));
         tableDefinitions.put(InformationColumnsTableInfo.IDENT, new StaticTableDefinition<>(
             informationSchemaIterables::columns,
```

Comment:
Not filtering early in ` informationSchemaIterables::columns` because it's used for `pg_attribute` where we need to expose dropped columns.